benchmarks/Microsoft.AspNetCore.Routing.Performance/Matchers/MatcherSelectCandidatesSmallEntryCountBenchmark.cs

```diff
@@ -72,43 +68,31 @@ private Matcher SetupMatcher(MatcherBuilder builder)
         }
 
         [Benchmark(Baseline = true)]
-        public async Task Baseline()
+        public unsafe void Baseline()
```

Comment:
What is unsafe here?

Reply:
Doesn't stackalloc always require unsafe? I'll see what happens if I remove

Reply:
It turns out you're right btw. You don't need unsafe for stackalloc - you need unsafe to dereference a pointer.